java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -1772,6 +1777,34 @@ void testContainsVector() {
     }
   }
 
+  @Test
+  void testStringOpsEmpty() {
```

Comment:
Should there also be tests for passing a null Scalar to these string ops?

Reply:
Ah nevermind.  I see that there are existing tests, but we're updating the exception types expected to be thrown.

Reply:
Those fail with a `CdfException` instead of an AssertionError and are still in the same test functions that were there before.